modules/express/src/expressApp.ts

```diff
@@ -282,13 +283,13 @@ module.exports.app = function(cfg: Config) {
 
 module.exports.init = co(function *() {
```

Comment:
not sure how to change this to the Es6 format

Reply:
You can do this:
```
export function init() {
  return co(function*() {
    // you can use yield in here
  }).call(this);
}
```